src/Shared/test/Shared.Tests/Microsoft.AspNetCore.Shared.Tests.csproj

```diff
@@ -18,13 +18,15 @@
     <Compile Include="$(SharedSourceRoot)HttpSys\**\*.cs" Link="Shared\HttpSys\%(Filename)%(Extension)"/>
     <Compile Include="$(SharedSourceRoot)NonCapturingTimer\*.cs" Link="Shared\NonCapturingTimer\%(Filename)%(Extension)"/>
     <Compile Include="$(SharedSourceRoot)ObjectMethodExecutor\*.cs" Link="Shared\ObjectMethodExecutor\%(Filename)%(Extension)"/>
+    <Compile Include="$(SharedSourceRoot)ParameterDefaultValue\*.cs" Link="Shared\ParameterDefaultValue\%(Filename)%(Extension)"/>
```

Comment:
Why is this needed?

Reply:
Not needed, it was mistakenly included in the merge between extensions and aspnetcore during migration.

Reply:
Correction, it is needed to compile ActivatorUtilities tests.